addons/payment/views/payment_icon_views.xml

```diff
@@ -21,6 +21,17 @@
         </field>
     </record>
 
+    <record id="payment_icon_form_tree" model="ir.ui.view">
+        <field name="name">payment.icon.tree</field>
+        <field name="model">payment.icon</field>
+        <field name="arch" type="xml">
+            <tree string="Payment Icons">
```

Comment:
```suggestion
            <tree>
```
IIRC, the name is the name of the actions, specifying the string on the tree tag has no impact anymore :thinking: